master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE checkpoints_v2 
+    ALTER COLUMN id DROP DEFAULT;
+
+DROP FUNCTION best_checkpoint_by_metric;
+DROP FUNCTION experiments_best_checkpoints_by_metric;
```

Comment:
These functions were created under the impression that checkpoints were attached to trials. Since `checkpoints_v2` checkpoints are generic, I didn't replace them.